variables.tf

```diff
@@ -85,10 +85,10 @@ variable "aws_guardduty_s3_protection" {
   description = "Whether AWS GuardDuty S3 protection should be enabled"
 }
 
-variable "aws_region_restrictions" {
+variable "aws_root_scp_exceptions" {
```

Comment:
Since you are changing the name lets use this opportunity to make this part a bit better. We have a lot of variables controlling the SCP's:

aws_deny_disabling_security_hub
aws_deny_leaving_org
aws_deny_root_user_ous
aws_require_imdsv2
aws_region_restrictions

lets combine these in 1 var called 
variable "aws_service_control_policies" {

then you can set the principal_exceptions for the security_hub and leaving_org var.

Reply:
Like this?
variable "aws_service_control_policies" {
  type = object({
    allowed_regions                 = optional(list(string))
    principal_exceptions            = optional(list(string))
    aws_deny_disabling_security_hub = optional(bool)
    aws_deny_leaving_org            = optional(bool)
    aws_deny_root_user_ous          = optional(list(string))
    aws_require_imdsv2              = optional(bool)
  })
}